tests/unit/admin/mapper/test_site_readings.py

```diff
@@ -0,0 +1,304 @@
+from decimal import Decimal
+from zoneinfo import ZoneInfo
+import pytest
+from datetime import datetime
+from envoy.admin.mapper.site_reading import AdminSiteReadingMapper
+from envoy.server.model.site_reading import SiteReading, SiteReadingType
+from assertical.fake.generator import generate_class_instance
+from envoy_schema.admin.schema.site_reading import (
+    CSIPAusSiteReading,
+    CSIPAusSiteReadingPageResponse,
+    CSIPAusSiteReadingUnit,
+    PhaseEnum,
+)
+from envoy_schema.server.schema.sep2.types import FlowDirectionType
+
+TZ = ZoneInfo("Australia/Brisbane")
+
+
+class MockSiteReadingType:
+    """Mock SiteReadingType for testing"""
+
+    def __init__(self, power_of_ten_multiplier: int, flow_direction: FlowDirectionType, phase: int):
+        self.power_of_ten_multiplier = power_of_ten_multiplier
+        self.flow_direction = flow_direction
+        self.phase = phase
+
+
+class MockSiteReading:
+    """Mock SiteReading for testing"""
+
+    def __init__(
+        self, time_period_start: datetime, time_period_seconds: int, value: int, site_reading_type: MockSiteReadingType
+    ):
+        self.time_period_start = time_period_start
+        self.time_period_seconds = time_period_seconds
+        self.value = value
+        self.site_reading_type = site_reading_type
+
+
+def test_csip_unit_to_uom_mapping():
+    """Test CSIP unit to UOM conversion for all valid units"""
+    assert AdminSiteReadingMapper.csip_unit_to_uom(CSIPAusSiteReadingUnit.ACTIVEPOWER) == 38
```

Comment:
A better test here might be something like the below - mainly to make sure you're covered for all future values/changes:
```
def test_csip_unit_to_uom_mapping():
  mapped_uoms = []
  for reading_unit in CSIPAusSiteReadingUnit:
    uom = AdminSiteReadingMapper.csip_unit_to_uom(reading_unit)
    assert isinstance(uom, UomType)
    mapped_uoms.append(uom)
  assert len(mapped_uoms) == len(set(mapped_uoms)), "Every mapping should be unique"
```